tests/test_abv_template.py

```diff
@@ -0,0 +1,353 @@
+# Note: testcases generated via `python -m cxxheaderparser.gentest`
```

Comment:
I wonder if it'd be worth using one of the popular pytest snapshot testing plugins for these. These tests look a bit gnarly.

Reply:
However, if you think it's worth going down that road, maybe write a test or two using that framework and let's take a look?